packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit.tsx

```diff
@@ -1,5 +1,302 @@
-import { FC } from 'react';
+import React, { FC, useEffect } from 'react';
+import {
+  Item,
+  ModalRow,
+  ModalLabel,
+  Grid,
+  useTranslation,
+  BasicTextInput,
+  Divider,
+  Table,
+  Fab,
+  TableCell,
+  TableCellProps,
+  TableContainer,
+  TableHead,
+  TableRow,
+  PlusCircleIcon,
+  generateUUID,
+  NumericTextInput,
+  formatDate,
+} from '@openmsupply-client/common';
+import { InboundShipmentItem, InboundShipmentRow } from '../../../types';
+import { ItemSearchInput } from '@openmsupply-client/system';
+import { flattenInboundItems } from '../../../utils';
+interface InboundLineEditProps {
+  item: InboundShipmentItem | null;
+  onUpsert: (item: InboundShipmentItem) => void;
+  onChangeItem: (item: Item | null) => void;
+}
 
-export const InboundLineEdit: FC = () => {
-  return null;
+const BasicCell: React.FC<TableCellProps> = ({ sx, ...props }) => (
```

Comment:
I'm thinking that a bunch of these components will be useful elsewhere and we may have to lift them out to `common`

Reply:
yup for sure!

I was actually wondering if we could try and use the `DataTable` also..